Movie/Movie/KeychainService/KeychainService.swift

```diff
@@ -0,0 +1,48 @@
+// KeychainService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Сервис для сохранения зашифрованной информации.
+final class KeychainService {
+    // MARK: - Public method
+
+    static func updateAPI(key: String) {
```

Comment:
```suggestion
    func updateAPI(key: String) {
```
static старайся не юзать объекты через конструктор прокидывай